sites/public/pages/eligibility/age.tsx

```diff
@@ -50,6 +50,16 @@ const EligibilityAge = () => {
     eligibilityRequirements.setCompletedSections(CURRENT_PAGE + 1)
   }
 
+  const onClick = (data) => {
+    if (isAgeValid(data.age)) {
+      eligibilityRequirements.setAge(data.age)
+      void router.push(eligibilityRoute(CURRENT_PAGE + 1))
+    } else {
+      setError("age", { type: "manual", message: "" })
```

Comment:
Do we want to set an error here? It looks like we're going to navigate to the filtered listings page anyway

Reply:
I realized I had some doubles in here from the onSubmit it was based on.  Refreshed here.

Reply:
Okay I think I'm understanding: the error message I'm seeing (at the bottom of this comment), isn't actually being generated by this line of code, right? Instead, that error message is coming from `onSubmit` on the `Form`, I think?

If that's all true, then I think we can get rid of this `else` altogether, since it seems like it's not doing anything?

![image](https://user-images.githubusercontent.com/8754454/135115665-69bccc28-93c3-4279-b568-7ea46bc7ea17.png)